fw/rbcx-coprocessor/src/ControlLink.cpp

```diff
@@ -9,22 +9,17 @@
 
 #include "Bsp.hpp"
 #include "ByteFifo.hpp"
-#include "cobs.h"
+#include "coproc_codec.h"
+#include "coproc_link_parser.h"
+#include "rbcx.pb.h"
 #include <array>
 
-enum class RxState {
-    AwaitingStart,
-    AwaitingLength,
-    ReceivingFrame,
-} static rxState;
-
 static DMA_HandleTypeDef dmaRxHandle;
 static DMA_HandleTypeDef dmaTxHandle;
 static ByteFifo<512> rxFifo;
-static int rxFrameLength = 0;
-static int rxFrameIndex = 0;
-static std::array<uint8_t, 255> rxFrameBuf;
 static std::array<uint8_t, 257> txFrameBuf;
```

Comment:
Bylo by fancy hodit to `257` do te comm knihovny jako konstantu, něco jako `MAX_ENCODE_BUFFER_SIZE`, ale je to maličkost.

Reply:
ok, jako makro?

Reply:
uf, já nevím, jak se teď dělají konstanty v c++, `static constexpr size_t`?

Reply:
Právě netuším, proto se ptám

Reply:
Tak asi ten constexpr, ať to má i typ, a dovnitř té `CoprocCodec` třídy by to dávalo smysl asi.

Reply:
Je tak